### PR TITLE
CMR-6134: More changes to insert and searching via ES 7.5.2 API.

### PR DESCRIPTION
**Pull request by ygliuvt**
With this PR, dev-system starts up fully without error.

### CODE REVIEW

File: elastic-utils-lib/src/cmr/elastic_utils/es_helper.clj [hunk 1]
```diff
@@ -26,3 +28,62 @@
...
+     (if (not-found? result)
+       nil
+       result))))
+
+(defn put
+  "Creates or updates a document in the search index, using the provided document id"
+  ([conn index mapping-type id document]
+   (put conn index mapping-type id document nil))
+  ([conn index mapping-type id document opts]
+   (if (empty? opts)
```
Comment (CayvonH) [inline elastic-utils-lib/src/cmr/elastic_utils/es_helper.clj]: i dont really understand why we need this check and can't just pass the nil opts to `query-params` but if it works i won't complain

File: indexer-app/src/cmr/indexer/data/index_set_elasticsearch.clj [hunk 2]
```diff
@@ -63,40 +65,47 @@
...
 (defn get-index-sets
   "Fetch all index-sets in elastic."
   [{:keys [conn]} index-name idx-mapping-type]
-  (when (esi/exists? conn index-name)
-    (let [result (doc/search conn index-name idx-mapping-type {"fields" "index-set-request"})]
-      (map (comp #(decode-field (first % )) :index-set-request :fields) (get-in result [:hits :hits])))))
+  (when (esi-helper/exists? conn index-name)
+    (let [result (es-helper/search
+                  conn index-name idx-mapping-type {"_source" "index-set-request"})]
+      (map (comp #(decode-field (first % )) :index-set-request :_source) (get-in result [:hits :hits])))))
```
Comment (CayvonH) [inline indexer-app/src/cmr/indexer/data/index_set_elasticsearch.clj]: nit: extra space in `(first % )`

File: elastic-utils-lib/src/cmr/elastic_utils/es_helper.clj (hunk 1 above)
Comment (ygliuvt) [reply elastic-utils-lib/src/cmr/elastic_utils/es_helper.clj]: I added this to solve a NullPointerException, but I don't think it is the cause of that. I will remove this check.

File: indexer-app/src/cmr/indexer/data/index_set_elasticsearch.clj (hunk 2 above)
Comment (ygliuvt) [reply indexer-app/src/cmr/indexer/data/index_set_elasticsearch.clj]: Done.

**Pull request merged**